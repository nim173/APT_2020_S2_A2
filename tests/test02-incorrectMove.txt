BURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLYBURLY
A
B
TURN 2 B 1
TURN 9 B 1