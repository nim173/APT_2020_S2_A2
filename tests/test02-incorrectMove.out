
Error: Tilebag is too small, should be atleast 100
